### DISCUSSION TITLE
希望增加直接进入直播功能

### DISCUSSION THREAD

**Discussion by frankmumu** [community/community#57732] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

希望增加启动应用直接进入直播界面的功能，方便老人看直播，谢谢

**Comment by queenofcorgis** [comment]
Hi @frankmumu , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
